src/rc_app.erl

```diff
@@ -12,3 +15,6 @@ start(_StartType, _StartArgs) ->
 
 stop(_State) ->
   ok.
+
+get_connection(ConnName) ->
```

Comment:
this should leave in a reconnections.erl module, so the api user could use it like `reconnections:get(ConnName)`. Also I think in that case`get` would be better than `get_connection`, because `reconnections:get_connection` would be kind of redundant,

Reply:
Agree.